pkg/resourceinterpreter/predefinedconfigurableinterpreter/configurable.go

```diff
@@ -0,0 +1,185 @@
+package predefinedconfigurableinterpreter
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	configv1alpha1 "github.com/karmada-io/karmada/pkg/apis/config/v1alpha1"
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+	"github.com/karmada-io/karmada/resourcecustomizations"
+)
+
+const configurableInterpreterFile = "customizations.lua"
```

Comment:
That is to say, the script filename is fixed, and can't write in multiple files?